src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -1,7 +1,5 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-#nullable enable
```

Comment:
out of curiosity, why?

Reply:
@tmat this commit (bdd592c8036b4e5d00c4451dec5a424fe8a41038) appears to be largely incorrect

Reply:
Removed. Not sure how that happened.